docs/keto/sdk/go.md

```diff
@@ -0,0 +1,231 @@
+---
+id: go
+title: Go
+---
+
+import CodeBlock from '@theme/CodeBlock'
+import { useLatestRelease } from '@site/src/hooks'
+
+
+Ory Keto exposes two APIs for integration
+
+- [gRPC](http://localhost:3001/docs/keto/reference/proto-api)
+- [REST](http://localhost:3001/docs/keto/reference/rest-api)
+
+## Installing SDK
```

Comment:
```suggestion
## Installation
```

Let's keep the same structure as in the other SDK guides, e.g. [hydra go](https://www.ory.sh/docs/hydra/sdk/go)